File: UNIDAD 3/JAVA/U3_P8.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3_p8;
import java.util.Scanner;
/**
 *
 * @author fatim
 */
public class U3_P8 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int a,b;
        System.out.print("Escribe numero 1: ");
        a = sc.nextInt();
        System.out.print("Escribe numero 2: ");
        b = sc.nextInt();
        if (a>b){
            System.out.println("El numero " + a + " Es mayor que " + b);
        } else if (b>a) {
            System.out.println("El numero " + b + " Es mayor que " + a);
        } 
    }
}

File: UNIDAD 3/C#/U3_P8.cs
using System;
namespace U3_P8
{
    internal class U3_P8
    {
        static void Main(string[] args)
        {
            int a, b;
            Console.Write("Escribe numero 1: ");
            a = Convert.ToInt32(Console.ReadLine());
            Console.Write("Escribe numero 2: ");
            b = Convert.ToInt32(Console.ReadLine());
            if (a > b)
            {
                Console.WriteLine("El numero " + a + " Es mayor que " + b);
            }
            else if (b > a)
            {
                Console.WriteLine("El numero " + a + " Es menor que " + b);
            }
        }
    }
}